src/client/auth/signedUNChallenge.ts

```diff
@@ -1,18 +1,14 @@
-import { AbstractAuthentication, AuthenticationMethod, AuthenticationResult } from './abstractAuthentication'
+import { AbstractAuthentication, AuthenticationMethod, AuthenticationResult, AuthenticationMethodUN } from './abstractAuthentication'
 import { AuthenticateInterface, OffChainTokenConfig, OnChainTokenConfig } from '../interface'
 import { SafeConnectProvider } from '../../wallet/SafeConnectProvider'
 import { UN, UNInterface } from './util/UN'
 import { logger } from '../../utils'
 
-export class SignedUNChallenge extends AbstractAuthentication implements AuthenticationMethod {
+export class SignedUNChallenge extends AbstractAuthentication implements AuthenticationMethodUN {
 	TYPE = 'signedUN'
 	private static DEFAULT_ENDPOINT = 'https://attestation-verify.tokenscript.org/un'
```

Comment:
@nicktaras would be great to move DEFAULT_ENDPOINT to the default config, to stay all default values in single place.

Reply:
I agree, let's to streamline this PR / task move this to a Jira ticket within this EPIC.

I'll create a Jira ticket capturing this task to address.

Reply:
https://smarttokenlabs.atlassian.net/browse/TIKAT-292